modules/cli/pages/cbstats/cbstats-collections-details.adoc

```diff
@@ -22,89 +22,94 @@ The optional `vbucket-reference` parameter is an integer, in the range of `0` to
 
 ----
-/opt/couchbase/bin/cbstats -u Administrator -p password -b testBucket \
-localhost:11210 collections-details
+/opt/couchbase/bin/cbstats localhost:11210 \
+-u Administrator -p password \
+-b travel-sample collections-details
 ----
 
 The initial section of the output is as follows:
 
 ----
-manifest:collection:0x0:name:                      _default
-manifest:collection:0x8:name:                      my_collection_in_my_scope_1
-manifest:collection:0x9:name:                      my_collection_in_my_scope_2
-manifest:collection:0xa:name:                      my_collection_in_default_scope
-manifest:collections:                              4
-manifest:default_exists:                           true
-manifest:uid:                                      4
-vb_0:collection:0x0:entry:high_seqno:              0
-vb_0:collection:0x0:entry:items:                   0
-vb_0:collection:0x0:entry:persisted_high_seqno:    0
-vb_0:collection:0x0:entry:scope:                   0
-vb_0:collection:0x0:entry:start_seqno:             0
-vb_0:collection:0x8:entry:high_seqno:              2
-vb_0:collection:0x8:entry:items:                   0
-vb_0:collection:0x8:entry:persisted_high_seqno:    2
-vb_0:collection:0x8:entry:scope:                   8
-vb_0:collection:0x8:entry:start_seqno:             2
-vb_0:collection:0x9:entry:high_seqno:              3
-vb_0:collection:0x9:entry:items:                   0
+0x0:0x0:name:                     _default
+0x8:0x9:name:                     MyCollection
+manifest_uid:                     4
+vb_0:0x0:high_seqno:              38
+vb_0:0x0:items:                   38
+vb_0:0x0:persisted_high_seqno:    38
+vb_0:0x0:scope:                   0x0
+vb_0:0x0:start_seqno:             0
+vb_0:0x9:high_seqno:              41
+vb_0:0x9:items:                   0
+vb_0:0x9:persisted_high_seqno:    41
+vb_0:0x9:scope:                   0x8
+vb_0:0x9:start_seqno:             41
+vb_0:collections:                 2
+vb_0:manifest:uid:                4
+vb_1:0x0:high_seqno:              29
+vb_1:0x0:items:                   29
+vb_1:0x0:persisted_high_seqno:    29
       .
       .
       .
 ----
 
-The initial seven lines provide information that identifies existing collections.
-A total of `4` are listed, including the `_default` collection.
-Each is identified with a hexadecimal number: for example, the `_default` collection is identified as `0x0`.
+Each line of the output presents a data _key_.
+Each key is of the format `scopeID` &#58; `collectionID` &#58; `datumLabel`.
+Each `scopeID` or `collectionID` is a hexadecimal number, prefixed by `0x`.
+Each `datumLabel` is a string.
+Each data key is concluded with a colon, and the associated value is presented to the right of the colon.
 
-Subsequent information in the output relates to each vBucket corresponding to `testBucket`.
+The initial two lines provide information that identifies existing collections.
+Two are listed, which are the `_default` collection, and `MyCollection`.
```

Comment:
Okay. I may have to be a bit vague about this at present, because the rbac stuff doesn't yet seem to be fully established, so it's hard for me to test. Unless you know the list of roles that will allow inspection? (RO, Cluster, Bucket, etc).